code-challenges.js

```diff
@@ -12,16 +12,42 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
-
-
-
+// write a funciton that takes a number as an argument and checks if it is % 3
+// if %3 === 0 return "() is divisible by three"
+//if  %3 !== 0 return "() is not divisible by three"
+// use else as instructions
+// revision-- added checker
+const threeFinder = (num) => {
+
+  if (typeof num !== "number"){
+    return "please enter a number"
+  }
+
+  else if (num %3 === 0){
+    return (num) + " is divisible by three"}
+  else  {
+    return (num) + " is not divisible by three"
+  }
+
+}
+console.log(threeFinder(num1));
+console.log(threeFinder(num2));
+console.log(threeFinder(num3));
+console.log(threeFinder("igda"));
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+//write a function named capitalizer that uses .toUpperCase after mapping through the array
+const capitalizer = (array) => {
+  let nounsMap = array.map (randomNoun => randomNoun.charAt(0).toUpperCase() + randomNoun.slice(1))
+  return nounsMap
+}
```

Comment:
yay! you can also return this map outright if you'd like, instead of creating a new variable!